docs/General/loginToRPS.md

```diff
@@ -45,11 +25,12 @@ The web portal is available for login after the deployment of the MPS, RPS, and
     | **Username**| standalone |
     | **Password**| G@ppm0ym |
 
-3. The home page should look like the following.
+4. The home page should look like the following.
```

Comment:
The home page is show below in Figure 1.

Reply:
Updated.